### PR TITLE
Update go mod and add test for schedule webhook

### PR DESCRIPTION
**Pull request by AsterNighT**
Signed-off-by: AsterNighT <klxjt99@outlook.com>

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: api/v1alpha1/schedule_webhook_test.go [hunk 1]
```diff
@@ -0,0 +1,101 @@
+// Copyright 2020 Chaos Mesh Authors.
```
Comment (fewdan) [inline api/v1alpha1/schedule_webhook_test.go:1]: s/2020/2021/g

**Review by fewdan — COMMENTED** [review]
rest LGTM

File: api/v1alpha1/schedule_webhook_test.go [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+								Action: ContainerKillAction,
+							}}},
+							Type:     ScheduleTypePodChaos,
+							Schedule: "@every 5s",
+						},
+					},
+					execute: func(schedule *Schedule) error {
+						return schedule.ValidateCreate()
+					},
+					expect: "error",
```
Comment (cwen0) [inline api/v1alpha1/schedule_webhook_test.go:51]: Why `"@every 5s"` is invalid?

Comment (YangKeao) [reply api/v1alpha1/schedule_webhook_test.go:51]: `containerNames` shouldn't be empty if the `action` is "container-kill"

**Review by YangKeao — APPROVED** [review]
LGTM

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**